vignettes/estimate_infections.Rmd

```diff
@@ -77,4 +126,100 @@ Beyond the forecast horizon ($T$), by default, the Gaussian process is assumed t
 
 where $I^c_t = \sum_{s< t} I_s$ are cumulative infections by $t-1$ and $I'_t$ are the unadjusted infections defined above. This adjustment is based on the one implemented in the `epidemia` R package [@bhattSemiMechanisticBayesianModeling].
 
+### Gaussian Process implementation details
+
+The Gaussian Processes labelled $\mathrm{GP}_t$ above can be written as
+
+\begin{equation}
+\mathcal{GP}(\mu(t), k(t, t'))
+\end{equation}
+
+where $\mu(t)$ and $k(t,t')$ are the mean and covariance functions, respectively.
+In our case as set out above, we have
+
+\begin{equation}
+\mu(t) \equiv 0 \\
+k(t,t') = k(|t - t'|) = k(\Delta t)
+\end{equation}
+
+where by default $k$ is a Matern 3/2 covariance kernel,
+
+\begin{equation}
+k(\Delta t) = \alpha \left( 1 + \frac{\sqrt{3} \Delta t}{l} \right) \exp \left( - \frac{\sqrt{3} \Delta t}{l}\right)
+\end{equation}
+
+with $l>0$ and $\alpha > 0$ the length scale and magnitude, respectively, of the kernel.
+Alternatively, a squared exponential kernel can be chosen,
+
+\begin{equation}
+k(\Delta t) = \alpha \exp \left( - \frac{1}{2} \frac{(\Delta t^2)}{l^2} \right)
+\end{equation}
+
+We use an Hilbert space approximation to the Gaussian Process [@approxGP], centered around mean zero.
+
+\begin{equation}
+\mathcal{GP}(0, k(\Delta t)) \approx \sum_{j=1}^m \left(S_k(\sqrt{\lambda_j}) \right)^\frac{1}{2} \phi_j(t) \beta_j
+\end{equation}
+
+with $m$ the number of basis functions to use in the approximation, which we calculate from the number of time points $t_\mathrm{GP}$ to which the Gaussian Process is being applied (rounded up to give an integer value)
+
+\begin{equation}
+m = b t_\mathrm{GP}
+\end{equation}
+
+and values of $\lambda_j$ given by
+
+\begin{equation}
+\lambda_j = \left( \frac{j \pi}{2 L} \right)^2
+\end{equation}
+
+where $L$ is a positive number termed boundary condition, and $\beta_{j}$ are regression weights with standard normal prior
```

Comment:
which we set by default to 2

Reply:
I think it's 1.5 (listed at the end of the document) https://github.com/epiforecasts/EpiNow2/blob/f4b836e6f69d4c99c5afdba79cdb2c630b3705d5/R/opts.R#L392

Reply:
oops